content/docs/reference/version-policy.md

```diff
@@ -0,0 +1,262 @@
+          tuning: Katib</a>
+          (<a href="https://github.com/kubeflow/katib">GitHub</a>)
+          </td>
+        <td>Beta</td>
+        <td>v1alpha3</td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/serving/kfserving/">KFServing</a>
+          (<a href="https://github.com/kubeflow/kfserving">GitHub</a>)
+        </td>
+        <td>Beta</td>
+        <td>v0.2.2</td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/misc/metadata/">Metadata</a>
+          (<a href="https://github.com/kubeflow/metadata">GitHub</a>)
+        </td>
+        <td>Beta</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/training/mpi/">MPI training: MPI 
+          operator</a>
+          (<a href="https://github.com/kubeflow/mpi-operator">GitHub</a>)
+        </td>
+        <td>Alpha</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/training/mxnet/">MXNet training: MXNet 
+          operator</a>
+          (<a href="https://github.com/kubeflow/mxnet-operator">GitHub</a>)
+        </td>
+        <td>Alpha</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="/docs/notebooks/why-use-jupyter-notebook/">Notebook web
+          app</a>
+          (<a href="https://github.com/kubeflow/kubeflow/tree/master/components/jupyter-web-app">GitHub</a>)
+        <td>Stable</td>
+        <td>1.0.0</td>
+      </tr>
+      <tr>
+        <td><a href="/docs/notebooks/why-use-jupyter-notebook/">Notebook 
+          controller</a> 
+          (<a href="https://github.com/kubeflow/kubeflow/tree/master/components/notebook-controller">GitHub</a>)
+        </td>
+        <td>Stable</td>
+        <td>1.0.0</td>
+      </tr>
+      <tr>
+        <td><a href="/docs/pipelines/overview/pipelines-overview/">Pipelines</a>
+          (<a href="https://github.com/kubeflow/pipelines">GitHub</a>)
+        </td>
+        <td>Beta</td>
+        <td>0.1.40</td>
+      </tr>
+      <tr>
+        <td><a href="/docs/other-guides/multi-user-overview/">Profile 
+          Controller for multi-user isolation</a> 
+          (<a href="https://github.com/kubeflow/kubeflow/tree/master/components/profile-controller">GitHub</a>)
+        </td>
+        <td>Stable</td>
+        <td>1.0.0</td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/training/pytorch/">PyTorch training: PyTorch operator</a> 
+          (<a href="https://github.com/kubeflow/pytorch-operator">GitHub</a>)
+        </td>
+        <td>Stable</td>
+        <td>1.0.0</td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/serving/tfbatchpredict/">TensorFlow 
+          batch prediction</a>
+          (<a href="https://github.com/kubeflow/batch-predict">GitHub</a>)
+        </td>
+        <td>Alpha</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="/docs/components/serving/tfserving_new/">TensorFlow
+          serving</a></td>
+        <td>Stable</td>
+        <td></td>
```

Comment:
/cc @jlewi Can we put 1.0.0 here for TFServing?

Reply:
@abhi-g I would suggest removing tfbatchpredict and tfserving.

TFServing isn't really a Kubeflow component. There isn't much we are adding to TFServing. 

Batch predict isn't being maintained right now. The docs
https://www.kubeflow.org/docs/components/serving/tfbatchpredict/
say as much.

And the last commit to https://www.kubeflow.org/docs/components/serving/tfbatchpredict/ is over a year ago.

Reply:
Perfectly fine to remove Batch prediction row. 

So KFServing remains our only serving component that we advertise here in this list and its Beta. Is that acceptable?

Reply:
@abhi-g @jlewi we should lead to tfserving through KFServing

cc @ellis-bigelow

Reply:
Thanks all. I've removed the line entries for TFServing and TF Batch Prediction. Our KFServing page does mention **TensorFlow** though not TFServing. And our serving overview explains:

> Kubeflow supports two model serving systems that allow multi-framework model serving: KFServing and Seldon Core. Alternatively, you can use a standalone model serving system. This page gives an overview of the options, so that you can choose the framework that best supports your model serving requirements.

https://master.kubeflow.org/docs/components/serving/overview/